ipn/ipnstate/ipnstate.go

```diff
@@ -108,8 +108,12 @@ type PeerStatus struct {
 	OS        string // HostInfo.OS
 	UserID    tailcfg.UserID
 
+	Tags []string // Tags assigned to this node
+
 	TailscaleIPs []netaddr.IP // Tailscale IP(s) assigned to this node
 
+	PrimaryRoutes []netaddr.IPPrefix // Routes being routed via this node
```

Comment:
same. omitempty & elaborate. Does this include TailscaleIPs /32 and /128? Or only a subset of offered routes?

Reply:
Done. No they don't; described in comments.